### ISSUE TITLE
After upgrading to 2.057005, discover's jobs don't work anymore with Extreme Networks switches

### ISSUE THREAD

**Issue by Rodjeur70** [netdisco/netdisco#934]
## Expected Behavior
Discover's job for Extreme Networks switches worked fine until 2.0.52 version.

## Current Behavior
After upgrading App to 2.057005, discover for these switches doesn't work anymore. Still working for other vendors.
Failed with message:
`[3202340] 2022-09-30 07:59:46  info App::Netdisco version 2.057005 loaded.
[3202340] 2022-09-30 07:59:46  info discover: [10.124.216.31] started at Fri Sep 30 09:59:46 2022
[3202340] 2022-09-30 07:59:49  info discover: finished at Fri Sep 30 09:59:49 2022
[3202340] 2022-09-30 07:59:49  info discover: status error: discover cancelled: 10.124.216.31 failed to return valid os_ver`

## Possible Solution
No idea

## Steps to Reproduce (for bugs)
Launch a discover job upon Extreme Networks switches

## Context
Managing an heterogenous vendors network with Netdisco App

## Your Environment
* Netdisco version used: 2.0.57.
* SNMP::Info version used: 5.9

## Config info (deployment.yml)
No modification since last functional version

**Comment by ollyg** [comment]
Hi @Rodjeur70 

Please could you send the output of these two commands for the device with the problem:

````
~/bin/netdisco-do show -d x.x.x.x os_ver
~/bin/netdisco-do dumpconfig -e snmp_field_protection
````

**Comment by ollyg** [comment]
sorry there was an error in my example, I've edited the comment just now

**Comment by Rodjeur70** [comment]
Hi Ollyg,

Following, result for these commands:
~/bin/netdisco-do show -d 10.124.216.31 -e os_ver
`[3215116] 2022-09-30 09:09:42  info App::Netdisco version 2.057005 loaded.
[3215116] 2022-09-30 09:09:42  info show: [10.124.216.31]/os_ver started at Fri Sep 30 11:09:42 2022
undef
[3215116] 2022-09-30 09:09:42  info show: finished at Fri Sep 30 11:09:42 2022
[3215116] 2022-09-30 09:09:42  info show: status done: Showed os_ver response from 10.124.216.31`

~/bin/netdisco-do dumpconfig -d 10.124.216.31 -e snmp_field_protection
`{
    device   {
        serial   [
            [0] "group:__ANY__"
        ]
    }
}`

I've tried to add filed protection for os_ver like this:
`{
    device   {
        os_ver   [
            [0] "group:__ANY__"
        ],
        serial   [
            [0] "group:__ANY__"
        ]
    }
}`
Same result.
However, discover works fine removing "os_ver" into @properties array in file lib/perl5/App/Netdisco/Worker/Plugin/Discover/Properties.pm:
`my @properties = qw/
    snmp_ver
    description uptime name
    layers mac
    ps1_type ps2_type ps1_status ps2_status
    fan slots
    vendor os
  /;`

**Comment by ollyg** [comment]
Thanks @Rodjeur70 OK I have an idea where this is, a regression in the code. I'll work on it ASAP.

**Comment by ollyg** [comment]
Hi @Rodjeur70 I have released 2.057006 which should be on the mirrors in a couple of hours, with a fix for the regression. Sorry about that!

**Comment by Rodjeur70** [comment]
Hi Ollyg,

No problem, thx for fast action ! I've upgraded App saturday morning with last patched version, all works fine now.
Thanks a lot for all your job and your amazing software, I'm a fervent user since many years now.

Regards,

Rodjeur
